ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -106,6 +106,10 @@ func clusterUpFlags() []cli.Flag {
 			Usage: "[Optional] Specify the AMI ID for your container instances. Defaults to amazon-ecs-optimized AMI.",
 		},
 		cli.BoolFlag{
+			Name:  command.NoAutoAssignPublicIPAddress,
+			Usage: "[Optional] The new instance instances will not have an Auto-assigned public address. By default, new instances will have an Auto-assigned public IP adress",
```

Comment:
[Optional] Do not assign public IP addresses to new instances in this VPC. Unless this option is specified, new instances in this VPC receive an automatically assigned public IP address.